src/components/Questions.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import ListSubheader from "@mui/material/ListSubheader";
+import List from "@mui/material/List";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+import Collapse from "@mui/material/Collapse";
+import ExpandLess from "@mui/icons-material/ExpandLess";
+import ExpandMore from "@mui/icons-material/ExpandMore";
+import classes from "./Questions.module.css";
+
+export default function Questions() {
+  const [open, setOpen] = React.useState(false);
+  const [secondOpen, setSecondOpen] = React.useState(false);
+  const [thirdOpen, setThirdOpen] = React.useState(false);
+  const [fourOpen, setfourOpen] = React.useState(false);
+  const [fiveOpen, setfiveOpen] = React.useState(false);
+  const [sixOpen, setSixOpen] = React.useState(false);
+  const [sevenOpen, setSevenOpen] = React.useState(false);
+
+  const handleClick = () => {
+    setOpen(!open);
+  };
+  const handleClick_Two = () => {
+    setSecondOpen(!secondOpen);
+  };
+  const handleClick_Three = () => {
+    setThirdOpen(!thirdOpen);
+  };
+  const handleClick_Four = () => {
+    setfourOpen(!fourOpen);
+  };
+  const handleClick_Five = () => {
+    setfiveOpen(!fiveOpen);
+  };
+  const handleClick_Six = () => {
+    setSixOpen(!sixOpen);
+  };
+  const handleClick_Seven = () => {
+    setSevenOpen(!sevenOpen);
+  };
+
+  return (
+    <List
+      className={classes.list}
+      sx={{ width: "100%", maxWidth: 640 }}
+      component="nav"
+      aria-labelledby="nested-list-subheader"
+      subheader={
+        <ListSubheader component="div" className={classes.subheader}>
+          Vanliga Frågor och Svar
+        </ListSubheader>
+      }
+    >
+      <ListItemButton onClick={handleClick}>
+        <ListItemText primary="Hur länge innan föreställningen kan jag avboka min biljett?" />
+        {open ? <ExpandLess /> : <ExpandMore />}
+      </ListItemButton>
+      <Collapse in={open} timeout="auto" unmountOnExit>
+        <List component="div" disablePadding>
+          <ListItemButton sx={{ pl: 4 }}>
+            <ListItemText secondary="Senast 30 minuter innan förställningen startar går det att avboka biljett. Avbokning sker i kassan på biografen alt via telefon." />
+          </ListItemButton>
+        </List>
+      </Collapse>
```

Comment:
This pattern repeats over and over in this component. That's a great candidate for a separate component that could be used here.

I would move all of this code to a separate file, replace the text with properties, and then do something like this here:

```tsx
<MyCustomComponent
  question="Hur länge innan föreställningen kan jag avboka min biljett?"
  answer="Senast 30 minuter innan förställningen startar går det att avboka biljett. Avbokning sker i kassan på biografen alt via telefon."
/>
```

Reply:
aah, smart! :) I was so focused on MUI that i didn´t think of other ways of solving it..